hmpps-kotlin-spring-boot-autoconfigure/src/main/kotlin/uk/gov/justice/hmpps/kotlin/auth/HmppsAuthContextFacade.kt

```diff
@@ -0,0 +1,56 @@
+package uk.gov.justice.hmpps.kotlin.auth
+
+import org.springframework.security.core.Authentication
+import org.springframework.security.core.GrantedAuthority
+import org.springframework.security.core.context.SecurityContextHolder
+import org.springframework.stereotype.Component
+
+@Component
+class HmppsAuthContextFacade {
```

Comment:
copied from Prison API

Reply:
The name's a bit technical - I'm not sure I understand what a context facade actually is. Or maybe just needs a paragraph describing what the class is for?